### PR TITLE
Create 2024-Q3-SCI-WG.md

### PR DESCRIPTION
**Pull request by hepwori**
Create Q3 TAC check-in for SCI WG

### CODE REVIEW

File: TI-reports/2024/2024-Q3-SCI-WG.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+v1 (2023) focused on build integrity and provenance (“Build Track”).
+
+
+### Up Next
+
+v1.1 (soon!) with an updated threat model and minor clarifications
+
+Source Track in draft
+
+Hardware Attestations Track in development
```
Comment (marcelamelara) [inline TI-reports/2024/2024-Q3-SCI-WG.md:51]: Nit: We've evolved this into the Attested Build Environments Track. The PR to enter it into draft is currently open and under review.

**Review by marcelamelara — APPROVED** [review]
Thanks for this really thorough summary @hepwori !

File: TI-reports/2024/2024-Q3-SCI-WG.md (hunk 1 above)
Comment (hepwori) [reply TI-reports/2024/2024-Q3-SCI-WG.md]: Updated, thanks!

**Review by marcelamelara — APPROVED** [review]
Thanks for the update!

**Review by sevansdell — APPROVED** [review]
Great work. Glad to hear projects across TIs are collaborating! Having attestable, machine readable signals that can be used by enterprises about build is great.

**Review by lehors — APPROVED** [review]

**Review by steiza — APPROVED** [review]
Great update, thank you!

**Pull request merged**